.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -229,7 +229,7 @@ public static class DateTimeDefinitions
 		public const string MealTimeRegex = @"\b(at\s+)?(?<mealTime>lunchtime)\b";
 		public static readonly string NumberEndingPattern = $@"^(\s+(?<meeting>meeting|appointment|conference|call|skype call)\s+to\s+(?<newTime>{PeriodHourNumRegex}|{HourRegex})((\.)?$|(\.,|,|!|\?)))";
 		public const string OneOnOneRegex = @"\b(1\s*:\s*1)|(one (on )?one|one\s*-\s*one|one\s*:\s*one)\b";
-		public static readonly string LaterEarlyPeriodRegex = $@"\b({PrefixPeriodRegex})\s*\b\s*(?<suffix>{OneWordPeriodRegex})\b";
+		public static readonly string LaterEarlyPeriodRegex = $@"\b((({PrefixPeriodRegex})\s*\b\s*(?<suffix>{OneWordPeriodRegex}))|(eoy))\b";

```

Comment:
I don't think this is correct. We should probably leverage the same unspecificendofregex. 
For example, the current code doesn't cover "end of year".